### PR TITLE
Add ability to archive runs by batch

### PR DESCRIPTION
**Pull request by VkoHov**
Added `/batch-archive` API endpoint with `POST` method.

### CODE REVIEW

**Comment by VkoHov** [comment]
Resolves [#1185](https://github.com/aimhubio/aim/issues/1185)

File: aim/storage/structured/sql_engine/entities.py [hunk 1]
```diff
@@ -80,6 +80,15 @@ def find(cls, _id: str, **kwargs) -> Union[IRun, SafeNone]:
             return ModelMappedRun.from_model(model_obj, session)
         return SafeNone()
 
+    @classmethod
+    def find_some(cls, ids: List[str], **kwargs) -> List[IRun]:
+        session = kwargs.get('session')
+        if not session:
+            return SafeNone()
```
Comment (alberttorosyan) [inline aim/storage/structured/sql_engine/entities.py:87]: should be `return []`

File: aim/storage/structured/sql_engine/entities.py [hunk 2]
```diff
@@ -80,6 +80,15 @@ def find(cls, _id: str, **kwargs) -> Union[IRun, SafeNone]:
             return ModelMappedRun.from_model(model_obj, session)
         return SafeNone()
 
+    @classmethod
+    def find_some(cls, ids: List[str], **kwargs) -> List[IRun]:
```
Comment (alberttorosyan) [inline aim/storage/structured/sql_engine/entities.py:84]: can we rename to `find_many`?

File: aim/web/api/runs/views.py [hunk 3]
```diff
@@ -513,3 +515,19 @@ async def delete_runs_batch_api(runs_batch: RunsBatchIn):
     return {
         'status': 'OK'
     }
+
+
+@runs_router.post('/archive-batch/', response_model=StructuredRunsArchivedOut)
+async def archive_runs_batch_api(runs_batch: RunsBatchIn, run_in: StructuredRunsArchivedIn,
```
Comment (alberttorosyan) [inline aim/web/api/runs/views.py:521]: can we change this to pass archive flag as a query parameter? so the API will look like:
`POST /api/runs/archive-batch?archive=true|false`
and the body will be 
```
[
<run-hash-1>,
<run-hash-2>,
...
]
```

File: aim/web/api/runs/pydantic_models.py [hunk 4]
```diff
@@ -140,7 +140,7 @@ class StructuredRunUpdateIn(BaseModel):
 
 
 class StructuredRunsArchivedIn(BaseModel):
```
Comment (alberttorosyan) [inline aim/web/api/runs/pydantic_models.py:142]: looks like this class is not used anymore. please remove.

**Review by alberttorosyan — APPROVED** [review]
approved! just left comment for a minor issue. can be merged once fixed

File: aim/web/api/runs/pydantic_models.py (hunk 4 above)
Comment (VkoHov) [reply aim/web/api/runs/pydantic_models.py:142]: Ok, @alberttorosyan jan.

**Review by mihran113 — APPROVED** [review]
@VkoHov  looks good, just conflicts need to be resolved due to recent changes pushed to main for remote tracking

**Review by Hamik25 — APPROVED** [review]

**Pull request merged**